redaxo/src/addons/metainfo/lib/input.php

```diff
@@ -100,6 +108,8 @@ public function getAttributes()
 
     /**
      * Gibt alle Attribute in String-Form zurueck.
+     *
+     * @return string
      */
     public function getAttributeString()
```

Comment:
Hier fehlt html escaping in der implementierung?

Reply:
Scheint mir auch so.

Reply:
Separater PR

Reply:
https://github.com/redaxo/redaxo/pull/2853